doc/01_installation.md

```diff
@@ -215,6 +210,9 @@ Check that services are running:
 ---
 
 > If you don’t have an external PostGIS instance, **uncomment** the corresponding line in `docker-up.sh` to include the internal container.
+> In addition, you have to set `DBPATH_POSTGIS` env var to the host folder to mount the volume for the PostGIS data,
+> e.g. `export DBPATH_POSTGIS=/data/pg`. **IMPORTANT**: that folder has to have the right owner/permissions or the PostGIS container will
+> refuse to start (typically `sudo chown -R 999:999 ${BDPATH_POSTGIS}`, as 999 is the usual UID and GID for PostGIS user).
```

Comment:
I hit this problem when I tested this. Permission in PG data dir use to be a "pain in the ass" :). Thus, I think this explicit warning is useful.